elements/pf-table/pf-tr.ts

```diff
@@ -110,6 +110,8 @@ export class PfTr extends LitElement {
           <pf-button id="toggle-button"
                      aria-expanded=${String(this.expanded) as 'true' | 'false'}
                      plain
+                     role="button"
```

Comment:
We shouldn't need to set role here as this gets applied at element internals level if the variant is to any other variant (or lack of variant) other then value of `link` which sets to `none`.

```Suggestion
```

Reply:
Thank you for your suggestion.
 I have removed role="button" and pushed the changes.